### PR TITLE
dropzone

### PR DESCRIPTION
**Pull request by marudor**

### CODE REVIEW

File: definitions/dropzone_v4.x.x/flow_>=v0.23.x/dropzone_v4.x.x.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+type FilesToString = (files: File|Array<File>) => string;
```
Comment (jeffmo) [inline definitions/dropzone_v4.x.x/flow_>=v0.23.x/dropzone_v4.x.x.js]: namespace plz

**Comment by marudor** [comment]
@jeffmo mind explaining the travis error?

**Comment by jeffmo** [comment]
Weird, seems like an `npm install` failure? I restarted the build, let's see if it happens again...

**Comment by marudor** [comment]
Hmm werid. Well - lets hope this won't be common.

**Comment by jeffmo** [comment]
Merged in https://github.com/flowtype/flow-typed/commit/d02e5290a80489586472201bfa94f1927d289218

**Pull request closed without merging**